app-config-exec/src/index.test.ts

```diff
@@ -0,0 +1,120 @@
+import { defaultEnvExtensions, defaultExtensions, loadUnvalidatedConfig } from '@lcdev/app-config';
+import execParsingExtension from '.';
+
+const defaultOptions = {
+  environmentExtensions: defaultEnvExtensions().concat(execParsingExtension()),
+  parsingExtensions: defaultExtensions().concat(execParsingExtension()),
+};
+
+describe('execParsingExtension', () => {
+  it('reads from command as root level string', async () => {
+    process.env.APP_CONFIG = JSON.stringify({
+      $exec: 'printf test123',
```

Comment:
Will all of these pass on windows CI? (I think we only run it in `develop`)

Reply:
Good point, `printf` might not. Now that I've added `trimWhitespace` as default I can replace this with `echo`.